service_container/service_decoration.rst

```diff
@@ -313,4 +313,114 @@ The generated code will be the following::
 
     $this->services[Foo::class] = new Baz(new Bar(new Foo()));
 
+Control the Behavior When the Decorated Service Does Not Exist
+--------------------------------------------------------------
+
+.. versionadded:: 4.4
+
+    The ``decoration_on_invalid`` option has been introduced in Symfony 4.4.
```

Comment:
Thanks! Maybe we could add `In previous versions, an exception was always thrown.`

Reply:
What about:
```
In previous versions, a ``ServiceNotFoundException`` was always thrown.
```
?